python/doc/examples/functional_modeling/viscous_fall_field_function.ipynb

```diff
@@ -0,0 +1,353 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Define a function with a field output: the viscous free fall example"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example, we define a function which has a vector input and a field output. This is why we use the `PythonPointToFieldFunction` class to create the associated function and propagate the uncertainties through it."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction \n",
+    "\n",
+    "We consider an object inside a vertical cylinder which contains a viscous fluid. The fluid generates a drag force which limits the speed of the solid and we assume that the force depends linearily on the object speed:\n",
+    "\n",
+    "$$\n",
+    "m \\frac{dv}{dt} = - m g - c v\n",
+    "$$\n",
+    "\n",
+    "for any $t \\in [0, t_{max}]$ where:\n",
+    "\n",
+    "- $v$ is the speed $[m/s]$,\n",
+    "- $t$ is the time $[s]$,\n",
+    "- $t_{max}$ is the maximum time $[s]$,\n",
+    "- $g$ is the gravitational acceleration $[m.s^{-2}]$,\n",
+    "- $m$ is the mass $[kg]$,\n",
+    "- $c$ is the linear drag coefficient $[kg.s^{-1}]$.\n",
+    "\n",
+    "The previous differential equation has the exact solution:\n",
+    "\n",
+    "$$\n",
+    "z(t) = z_0 + v_{inf} t + \\tau (v_0 - v_{inf})\\left(1 - e^{-\\frac{t}{\\tau}}\\right)\n",
+    "$$\n",
+    "\n",
+    "for any $t \\in [0, t_{max}]$\n",
+    " \n",
+    "where:\n",
+    "\n",
+    "- $z$ is the altitude above the surface $[m]$,\n",
+    "- $z_0$ is the initial altitude $[m]$,\n",
+    "- $v_0$ is the initial speed (upward) $[m.s^{-1}]$,\n",
+    "- $v_{inf}$ is the limit speed $[m.s^{-1}]$:\n",
+    "\n",
+    "$$\n",
+    "v_{inf}=-\\frac{m g}{c}\n",
+    "$$\n",
+    "\n",
+    "- $\\tau$ is time caracteristic $[s]$:\n",
+    "\n",
+    "$$\n",
+    "\\tau=\\frac{m}{c}.\n",
+    "$$\n",
+    "\n",
+    "The stationnary speed limit at infinite time is equal to $v_{inf}$:\n",
+    "\n",
+    "$$\n",
+    "\\lim_{t\\rightarrow+\\infty} v(t)= v_{inf}. \n",
+    "$$\n",
+    "\n",
+    "When there is no drag, i.e. when $c=0$, the the trajectory depends quadratically on $t$:\n",
+    "\n",
+    "$$\n",
+    "z(t) = z_0 + v_0 t -g t^2\n",
+    "$$\n",
+    "\n",
+    "for any $t \\in [0, t_{max}]$.\n",
+    "\n",
+    "Furthermore, when the solid touches the ground, we ensure that the altitude remains nonnegative, i.e. the final altitude is:\n",
```

Comment:
```suggestion
    "Furthermore, when the solid touches the ground, we ensure that the altitude remains nonnegative, i.e. the final altitude is\n",
```
I feel there are too many breakpoints in this sentence. The colon and the comma on the next line (see next suggestion) could be dropped.

Reply:
Ok. Fixed (but less stylish !).